test/functional/manifest.ini

```diff
@@ -1,5 +1,7 @@
 [DEFAULT]
-run-if = buildapp == 'browser'
+# app == 'Firefox' is used for running in the Loop repo.
+# buildapp == 'browser' is used for mozilla-central & co.
+run-if = buildapp == 'browser' || app == 'Firefox'
```

Comment:
There's a separate manifest which lists the manifests for Mn tests - the functional test isn't one of those.
The buildapp is existing code, and just says run for the browser tests, as opposed to fennec etc.